tests/functional_tests/mining.py

```diff
@@ -95,7 +95,7 @@ def mine(self, via_daemon):
         assert res_status.block_reward >= 600000000000
 
         # wait till we mined a few of them
-        timeout = 60 # randomx is slow to init
+        timeout = 90 # randomx is slow to init
```

Comment:
The way this test is written makes it difficult to distinguish timeouts waiting for the initial block versus subsequent ones. It's possible (and I would say, likely) that the timeouts you see are actually for the subsequent blocks, and you should be changing `timeout = 5`, below, rather than this startup timeout. I also think the code would be much clearer (and easier to diagnose failures) if the init and subsequent block cases were separated; for example:
```
for _ in range(60):
    time.sleep(1)
    height = daemon.get_info().height
    if height > prev_height:
        break
else:
    assert False, 'Failed to mine initial block'

target_height = prev_height + 5
while height < target_height:
    for _ in range(5):
        time.sleep(1)
        height = daemon.get_info().height
        if height >= target_height:
            break
    else:
        assert False, 'Failed to mine successor to block %d' % (height,)
```

Reply:
Alternately, you could keep the single loop and just refactor it a bit, which may be superior:
```
while height < target_height:
    timeout = 60 if height == prev_height else 5
    cur_height = height
    for _ in range(timeout):
        time.sleep(1)
        height = daemon.get_info().height
        if height > cur_height:
            break
    else:
        assert False, 'Failed to mine successor to block %d' % (height,)
```

Reply:
Thank you, I committed your second suggestion.